src/app/components/XProfileWidget/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import './XProfileWidget.css'
+import XIcon from '@mui/icons-material/X'
+
+export const XProfileWidget: React.FC<{ handle: string }> = ({ handle }) => {
+  // Clean the handle (remove '@' if present)
+  const cleanHandle = handle.startsWith('@') ? handle.slice(1) : handle
+
+  // Construct X profile URL
+  const profileUrl = `https://x.com/${cleanHandle}`
+
+  // Construct profile picture URL using unavatar.io
+  const profilePicUrl = `https://unavatar.io/x/${cleanHandle}?fallback=https://via.placeholder.com/50`
+
+  return (
+    <a href={profileUrl} target="_blank" rel="noopener noreferrer" className="x-profile-widget">
+      <div className={'x-profile-link'}>
+        <div className="x-profile-card">
+          <img src={profilePicUrl} alt={`${cleanHandle}'s profile`} className="x-profile-picture" />
```

Comment:
we should probably load foreign images with crossorigin attr to reduce tracking

Reply:
Currently there is no test data, I always faked it locally, so there is no preview link. I am trying to create some ROFL app with the correct data to show... or in worst case I can push a commit with fake data later.

Reply:
Unfortunately, adding `crossorigin="anonymous"`  disables unavatar. The best I can do is `referrerpolicy="no-referrer"`,